intermediate_source/torchserve_aws_sagemaker_tutorial.rst

```diff
@@ -0,0 +1,232 @@
+TorchServe on AWS SageMaker
+============================
+
+In this tutorial you will learn how you can efficiently serve PyTorch models using Torchserve and AWS Sagemaker
+
+Why TorchServe?
+^^^^^^^^^^^^^^^^
+
+| TorchServe is the recommended model server for PyTorch, preinstalled in the AWS PyTorch Deep Learning Container (DLC). This powerful tool offers customers a consistent and user-friendly experience, delivering high performance in deploying multiple PyTorch models across various AWS instances, including CPU, GPU, Neuron, and Graviton, regardless of the model size or distribution.
+| TorchServe is easy to use. It comes with a convenient CLI to deploy locally and is easy to package into a container and scale out with Amazon SageMaker or Amazon EKS. With default handlers for common problems such as image classification, object detection, image segmentation, and text classification, you can deploy with just a few lines of code—no more writing lengthy service handlers for initialization, preprocessing, and post-processing. TorchServe is open-source, which means it's fully open and extensible to fit your deployment needs.
+
+To get started on how to use TorchServe you can refer to this tutorial: `TorchServe QuickStart  <https://pytorch.org/serve/getting_started.html>`_
+
```

Comment:
Please add the user manual links of using TorchServe on SM:
- https://docs.aws.amazon.com/sagemaker/latest/dg/deploy-models-frameworks-torchserve.html
- https://docs.aws.amazon.com/sagemaker/latest/dg/large-model-inference-tutorials-torchserve.html

Reply:
SageMaker has 2 different endpoints. Their deployment is slightly different. Please include this information.
- single model 
- multi-model

Reply:
> Please add the user manual links of using TorchServe on SM:
> 
>     * https://docs.aws.amazon.com/sagemaker/latest/dg/deploy-models-frameworks-torchserve.html
> 
>     * https://docs.aws.amazon.com/sagemaker/latest/dg/large-model-inference-tutorials-torchserve.html

Hello lxning,
Thanks for the review, I have already added these links in the reference at the end of the tutorial.